artemis-chart/files/configmaps/services.conf

```diff
@@ -0,0 +1,146 @@
+[inet_http_server]
```

Comment:
I think this does not require permanent storage since the user may change it but he needs to restart the backend. Think about whether we need it there or not.